migrations/20240501124220_not_null_token_values.sql

```diff
@@ -0,0 +1,25 @@
+-- +goose Up
+-- +goose StatementBegin
+SELECT 'up SQL query';
+
+ALTER TABLE rewards
+ALTER COLUMN streak_earnings SET DEFAULT 0,
```

Comment:
Ah crap I think you need to still do an `UPDATE rewards SET ...`. Trying to patch this up.